mp/src/game/server/player_lagcompensation.cpp

```diff
@@ -112,6 +114,7 @@ struct LagRecord
 	LayerRecord				m_layerRecords[MAX_LAYER_RECORDS];
 	int						m_masterSequence;
 	float					m_masterCycle;
+	float					m_poseParameters[MAX_POSE_PARAMETERS];
```

Comment:
You're probably going to want to add a `for` loop in the copy constructor to copy the `m_poseParameters` values over from `src` (similar to what's done for `m_layerRecords` in the copy ctor).